### PR TITLE
Init contributing guide

### PR DESCRIPTION
**Pull request by HichamELBSI**
- Init the contributing guide
- Upgrade version
- Update the root package.json scripts

### CODE REVIEW

**Review by ronronscelestes — APPROVED** [review]
LGTM 🚀

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Contribute to Strapi Design System
+
+The Strapi Design System is an open-source project administered by [the Strapi team](https://strapi.io/company). We appreciate your interest and efforts to contribute to the Design System.
+
+All efforts to contribute are highly appreciated, we recommend you talk to a maintainer prior to spending a lot of time making a pull request that may not align with the project roadmap.
```
Comment (soupette) [inline CONTRIBUTING.md:5]: > we recommend you talk to a maintainer prior to spending a lot of time making a pull request that may not align with the project roadmap.

Maybe we could add "by opening an issue..."

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## Code of Conduct
+
+This project and everyone participating in it are governed by the [Strapi Code of Conduct](https://github.com/strapi/strapi/blob/master/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please read the [full text](https://github.com/strapi/strapi/blob/master/CODE_OF_CONDUCT.md) so that you can read which actions may or may not be tolerated.
+
+## You found a bugs
+
+We are using [GitHub Issues](https://github.com/strapi/design-system/issues) to manage our public bugs. We keep a close eye on this so before filing a new issue, try to make sure the problem does not already exist. You can also use the [codesandbox template](https://codesandbox.io/s/strapi-design-system-r1vdp) to easily show us the bug.
```
Comment (soupette) [inline CONTRIBUTING.md:13]: In the codesandbox do we need to import the styles.css?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+```bash
+yarn storybook
+```
+
+#### 5. Start the website for documentation changes
+
+Start the DS website to test your changes on the documentation library.
+
+```bash
```
Comment (soupette) [inline CONTRIBUTING.md:69]: This command is the same as in step 4 is it wanted?

File: package.json [hunk 4]
```diff
@@ -14,14 +14,15 @@
...
     "specify:pull": "specify pull",
     "specify:sync": "specify sync",
     "storybook": "lerna run storybook --stream",
-    "test:example": "yarn --cwd example test --watchAll=false",
-    "test:e2e:ci": "lerna run test:e2e:ci",
     "test:e2e": "lerna run test:e2e --stream",
+    "test:e2e:ci": "lerna run test:e2e:ci",
+    "test:e2e:watch": "yarn --cwd packages/strapi-design-system test:e2e:watch",
     "test": "lerna run test --stream",
+    "test:watch": "yarn --cwd packages/strapi-design-system test:watch",
```
Comment (soupette) [inline package.json:24]: We should use lerna with the `--scope` flag instead.

Here's the [documentation](https://www.npmjs.com/package/@lerna/filter-options)

**Review by soupette — CHANGES_REQUESTED** [review]

**Review by soupette — APPROVED** [review]
LGTM! Good job

**Pull request merged**